src/index.jsx

```diff
@@ -101,7 +102,6 @@ class ReactAudioPlayer extends Component {
         ref={(ref) => { this.audioEl = ref; }}
         onPlay={this.onPlay}
         loop={this.props.loop}
-        {...this.props}
```

Comment:
Sorry, I meant that you should undo your change and keep this line (for now).  In another PR, I'll make the permanent fix, which is to explicitly support all audio tag attributes so that putting extra props onto the audio tag isn't necessary.  Then we can remove this line.